docs/lts_policy.rst

```diff
@@ -0,0 +1,21 @@
+*******************
+LTS Backport Policy
+*******************
+
+Starting from astropy 5.0, backports to Long-Term Stable (LTS) releases
+will only cover:
+
+* Critical security fixes.
+* Bug fixes where backporting is straightforward (e.g., no conflicts).
+* More complex fixes are permissible only as resources allow, ideally with
+  backports done by the original author or from institutions who are on LTS
```

Comment:
How about "users"? In most cases, those users are institutions, but I don't think that we should write a policy that explicitly excludes non-institutional users.

Reply:
I added "users". Look better now?